docs/reference/core_concepts/expectations/implemented_expectations.rst

```diff
@@ -10,110 +10,112 @@ for all platforms. This table details which are implemented. Note we love pull-r
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_max_to_be_between`                                                    | Y          | Y       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_chisquare_test_p_value_to_be_greater_than` *                          | Y          | Y       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_bootstrapped_ks_test_p_value_to_be_greater_than` *                    | Y          | N       | N         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_kl_divergence_to_be_less_than`                                        | Y          | Y       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_pair_values_to_be_equal` *                                            | Y          | N       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_pair_values_A_to_be_greater_than_B` *                                 | Y          | N       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_pair_values_to_be_in_set` *                                           | Y          | N       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_select_column_values_to_be_unique_within_record` *                           | Y          | N       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_compound_columns_to_be_unique` *                                             | Y          | Y       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_column_pair_cramers_phi_value_to_be_less_than` *                             | Y          | N       | N         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+|`expect_multicolumn_sum_to_equal` *                                                  | Y          | N       | Y         |
++-------------------------------------------------------------------------------------+------------+---------+-----------+
+
+`*` This expectation has not yet been migrated to the modular API.
```

Comment:
Hm looking at this in isolation, "modular API" seems a bit meaningless. Maybe say "the modular Expectations API (0.13)"

Reply:
Sounds good - I updated this phrasing for the docs page, and modified the tag to be clearer as well.